src/utils/findGitmojiCommand.ts

```diff
@@ -0,0 +1,86 @@
+// @flow
```

Comment:
```suggestion

```

Reply:
Prettier didn't run for this file it is full of `;` I believe CI will complain!

Reply:
```
Checking formatting...
All matched files use Prettier code style!
```

Has this been added as rule in prettier config? I don't see any prettierConfig as such.

Reply:
Add `lint-fix` command that autofixes any linter/prettier issues.